### ISSUE TITLE
Hotkeys disappears

### ISSUE THREAD

**Issue by arddhu-hub** [obsproject/obs-studio#3647]
## Platform
Operating system and version:  Windows 10 19041.572
OBS Studio version:  26.0.2

## Expected Behavior
Hotkeys should stay saved after close and open OBS.

## Current Behavior
When you create hotkeys for some Sources they work fine, but if you put those sources inside a group, when you close and open OBS again, the hotkey are not there and if you try to find the source (oon the Hotkey menus) to do the hotkey again the sources don't appear. You have to place the source ou of the group then restart OBS to be able to find it on the Hotkey interface again. 

## Steps to Reproduce
1. Create a Hotkey for a source 
2. Place the source inside a group
3. Reset OBS
4. Hotkey dissappear
5. Open Hotkey Interface
6. Look of the Source to apply Hotkey again
7. Don't find the source
8. Place the source out of the Group
9. Reset OBS
10. Open Hotkey Interface
12. Source Reappears

**Comment by WizardCM** [comment]
Could you confirm whether this issue occurs in previous versions?

**Comment by arddhu-hub** [comment]
> Could you confirm whether this issue occurs in previous versions?

Yes, it does, but I don't know the number of the versions.

**Comment by joshvector** [comment]
This happens for me. Using version 26, now 27, on macOS Big Sur and the issue happens on both versions. Exact same issue. Any word on a fix for this?

**Comment by PatTheMav** [comment]
Can't confirm this on macOS - hotkey entries are visible in the settings window even after relaunching OBS with the source placed into a group.

@Fenrirthviti didn't you confirm this off-thread this week?

**Comment by Fenrirthviti** [comment]
Correct, this happens on Windows and is 100% reproducible following the steps in the first post on 28.0.3 (on Windows 11, but I suspect version of Windows doesn't matter).

**Comment by SuslikV** [comment]
Here is video confirmation of the issue for OBS v30.0.0 recorded in VirtualBox environment.

CRC of the zip archive for "OBS-Studio-30.0.zip" application (149 688 544 bytes)
MD5:9AC5946123BB7EC0D162FA540DC7DA5D
SHA256:016D659A488DF07F386E8F2CE7A7DB977A788DB4AC65E6AF1119E326A76D44BF

I think, issue is the same or fix wasn't included in OBS v30.0.0, or there is other version of v30 of OBS....
Anyway, just in case:

https://github.com/obsproject/obs-studio/assets/19683044/f660c02c-13f6-417b-a7d1-3598d45191e0

**Comment by Fenrirthviti** [comment]
So, looks like this *was* fixed as presented with the original STR.

However, for some reason, if you use the context menu to "Group selected items" the old broken behavior as reported is present.

Going to reopen this so it is not forgotten about.

**Comment by prgmitchell** [comment]
Looking through old issues, can no longer repro this so it may be able to be closed @Fenrirthviti
